joomla/joomla-cms
# [4.0] Correcting wrong debug lang untranslated strings

### Summary of Changes 
Adding some missing labels (filter_contacts, filter_articles), correcting a typo (filter_banners), prevent double translation for tooltip (transition-button);

Note: I could not find why `COM_NEWSFEEDS_CATEGORIES` is noted as untranslated as the sys.ini is loaded...
### Testing Instructions

**Make sure https://github.com/joomla/joomla-cms/pull/27074 is merged or merge it first.**

Set debug lang on, show its subpage and navigate in the managers concerned above
Look at the Untranslated strings.

The menutypes are noted as Untranslated but this is expected (see explanation here: https://github.com/joomla/joomla-cms/issues/27068#issuecomment-554286371 )

`Actions` is marked as double translated, but we already have an issue for that and we were not able to find a solution.


### Before patch
Example when the Articles Manager is displayed

<img width="747" alt="Screen Shot 2019-11-15 at 12 21 12" src="https://user-images.githubusercontent.com/869724/68940063-79eea200-07a2-11ea-9312-c9815c82829e.png">


### After patch

<img width="760" alt="Screen Shot 2019-11-15 at 11 43 39" src="https://user-images.githubusercontent.com/869724/68939945-3300ac80-07a2-11ea-858f-2a4a2915d591.png">


### Documentation Changes Required
